lib/oppen.rb

```diff
@@ -12,13 +12,18 @@ module Oppen
   # Entry point of the pretty printer.
   #
   # @param config [Config]
+  # @param space [String, Proc] space string that will be translated into
+  #   a lambda of the form ->(n){ n * space } to generate spaces.
+  #   You can also directly pass a lambda.
```

Comment:
```suggestion
  # @param space [String, Proc] could be a {String} or a callable. 
  #   If it's a string, spaces will be generated with the the
  #   lambda `->(n){ n * space }`, where `n` is the number of columns
  #   to indent.
  #   If it's a callable, it will receive `n` and it needs to return
  #   a string.
```